Services/AuthentificationService/TokensStorage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AuthentificationService
+{
+    public static class TokensStorage
+    {
+        public static List<string> Tokens = new List<string>() { null };
```

Comment:
What's the reason of creating special class just for List? May be then it would be good to also add here methods like Add()/Remove() which we will just update as we change how tokens are stored?

Reply:
If you think this is right, then you can remake